Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -159,24 +159,13 @@ class ClearPrivateDataTableViewController: ThemedTableViewController {
                 }
             }
 
+            let alert: UIAlertController
             if self.toggles[HistoryClearableIndex] && profile.hasAccount() {
-                profile.syncManager.hasSyncedHistory().uponQueue(.main) { yes in
```

Comment:
I can bring this back, but this was a query for the purpose of determining if we should display:

"This action will clear all of your private data, including history from your synced devices"

or

"This action will clear all of your private data. It cannot be undone."

I removed it because the query would be complicated in the new history, and having the "including history from your synced devices" seems OK?